.github/docker-images/alpine-3.16-arm64/Dockerfile

```diff
@@ -18,7 +18,8 @@ RUN apk add \
     ca-certificates \
     cmake \
     bash \
-    aws-cli
+    aws-cli \
+    perl-strictures
```

Comment:
Why do we need this weird specific package?

https://metacpan.org/pod/strictures
> strictures - Turn on strict and make most warnings fatal

Is there something more basic, named like `perl`?

Reply:
yeah, but aws-lc needs strict module for perl.